desktop/cmp/chart/ChartModel.js

```diff
@@ -14,28 +14,23 @@ import {observable, action} from '@xh/hoist/mobx';
 @HoistModel
 export class ChartModel {
 
-    @observable.ref config = {};
-    @observable.ref series = [];
+    @bindable.ref highchartsConfig = {};
+    @bindable.ref series = [];
```

Comment:
Don't feel strongly about this, but explicit setters might still have some value if they are an important part of the public API.  WIll make it easier for tooling to find them, and if there are any caveats or notes to document (i.e. can you change anything about the highchartsConfig?) we would have a place to do so.